### PR TITLE
PLAT-126345: Add UI tests for agate/Panel

### PR DESCRIPTION
**Pull request by stanca-pop-lgp**
Enact-DCO-1.0-Signed-off-by: Stanca Pop stanca.pop@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
- added UI tests for agate/Panel

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
PLAT-126345

### Comments

### CODE REVIEW

**Review by adrian-cocoara-lgp — APPROVED** [review]

File: tests/ui/specs/Panel/Agate-Panel-specs.js [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+
+			Page.panel6.waitForEnter();
+
+			const expected = 'Panel6 Button 2';
+			const actual = Page.focusedText;
+
+			expect(actual).to.equal(expected);
+		});
+
+		// this test passed manually but fails in automation because the 'last-focused' element
```
Comment (daniel-stoian-lgp) [inline tests/ui/specs/Panel/Agate-Panel-specs.js:120]: does this comment apply here as well?

**Review by daniel-stoian-lgp — APPROVED** [review]

File: tests/ui/specs/Panel/Agate-Panel-specs.js (hunk 1 above)
Comment (stanca-pop-lgp) [reply tests/ui/specs/Panel/Agate-Panel-specs.js:120]: It does not, thank you.

File: tests/ui/specs/Panel/Agate-Panel-Page.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
+'use strict';
```
Comment (jeonghee27) [inline tests/ui/specs/Panel/Agate-Panel-Page.js:1]: you don't need to use '-' for Page file name.
Agate-PanelPage.js  is Better.
spec and view is mandatory for ui-test-utils. but Page is not

**Review by jeonghee27 — APPROVED** [review]
LGTM except file name

File: tests/ui/specs/Panel/Agate-Panel-Page.js (hunk 2 above)
Comment (stanca-pop-lgp) [reply tests/ui/specs/Panel/Agate-Panel-Page.js]: Ok, thank you!

**Review by MikyungKim — APPROVED** [review]
👍

**Pull request merged**